src/main/java/gov/nasa/cumulus/CNMResponse.java

```diff
@@ -349,10 +376,11 @@ public String PerformFunction(String input, Context context) throws Exception {
         // get collection and dataVersion for use in message attribute hash
         String collection = getCollection(inputKey);
         String dataVersion = getDataVersion(inputConfig);
+        String dataProcessingType = getDataProcessingType(inputKey);
```

Comment:
I think we want to pass inputConfig to getDataProcessingType and not inputKey.

https://github.com/podaac/cumulus-cnm-response-task/blob/develop/src/main/java/gov/nasa/cumulus/CNMResponse.java#L270

versus

https://github.com/podaac/cumulus-cnm-response-task/blob/develop/src/main/java/gov/nasa/cumulus/CNMResponse.java#L249

Not sure why the inconsistency is there in the first place.